aries-core/src/js/components/external/stories/MaskedInput.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { Box, Grommet, MaskedInput } from 'grommet';
+import { aries } from '../../../../../../aries-site/src/themes/aries';
+
+export default {
+  title: 'MaskedInput',
+};
+
+export const Email = () => {
+  const [value, setValue] = React.useState('');
+  return (
+    <Grommet theme={aries} full>
+      <Box fill align="center" justify="start" pad="large">
+        <Box width="medium">
```

Comment:
do we have to have this Box or can we leverage the parent Box instead?

Reply:
Good clean up, let me see.